README.md

```diff
@@ -225,20 +225,44 @@ Visit https://github.com/MikkelSchubert/adapterremoval/ to get the latest
 version number, and then:
 
 ```
-wget -O adapterremoval-2.3.1.tar.gz \
-     https://github.com/MikkelSchubert/adapterremoval/archive/v2.3.1.tar.gz
-tar xvzf adapterremoval-2.3.1.tar.gz
-cd adapterremoval-2.3.1
+wget -O adapterremoval-2.3.3.tar.gz \
+     https://github.com/MikkelSchubert/adapterremoval/archive/v2.3.3.tar.gz
+tar xvzf adapterremoval-2.3.3.tar.gz
+cd adapterremoval-2.3.3
 make
 sudo make install
 ```
 
+The adapter removal step downloads the fastq files one sample at a time from S3 to the local machine.
+The step also generates local output files there before copying them to S3.
+On Linux these files are stored in `/tmp`.
+If you don’t have enough space available in `/tmp`, AdapterRemoval will crash.
+To check the available space in `/tmp` run `df -H`.
+You should have at least 2x the size of your largest pair of fastq files.
+To resize `/tmp`, add this line to `/etc/fstab` (tab-separated, with whatever size you need):
```

Comment:
Probably instead of adding a line you're editing the line that's already there?

Reply:
At least on my instance there was no line there.

Reply:
This is what my `/etc/fstab` looked like by default on an AWS Linux 2023 instance:
```
#
UUID=9bbbebd1-cfa0-4f0c-b851-2dacbac7cf3c     /           xfs    defaults,noatime  1   1
UUID=68DE-EC82        /boot/efi       vfat    defaults,noatime,uid=0,gid=0,umask=0077,shortname=winnt,x-systemd.automount 0 2
```

Reply:
I could change the comment to suggest editing the line if it's there or adding a new one if not